src/test/java/org/agileandbeyond/dandd/tddexercise/Human.java

```diff
@@ -0,0 +1,8 @@
+package org.agileandbeyond.dandd.tddexercise;
+
+public class Human extends Race {
+
+	public Human() {
+		super();
```

Comment:
For readability, I would explicitly set the RaceType and the modifiers in the concrete class instead of relying on the defaults.